### ISSUE TITLE
 Just to get Clarity

### ISSUE THREAD

**Issue by mohananagavenkat** [aunyks/newton-api#9]
I just get into newton api and i want to know the special cases where this api is needed exactly, because all the math that this api does can be done easily by using javascript.

**Comment by aunyks** [comment]
Hi Mohana. The purpose of Newton is to abstract the math performed by the API for other developers. That is, Newton tries to make code cleaner for everyone performing these calculations. JavaScript developers aren't the only ones who can use Newton. It's a REST API so that it can be utilized by developers in a language-agnostic manner. As a matter of fact, anything that can make an HTTP request can use Newton! 

This is why in the README I've included wrapper libraries that people have created in different languages.

TL;DR:
It's not just for JS developers and it provides necessary abstraction for math.
